app/controllers/additional_submission_informations_controller.rb

```diff
@@ -0,0 +1,48 @@
+# voluntary submission information controller for creation only
+class AdditionalSubmissionInformationsController < ApplicationController
+  def new
+    @info = AdditionalSubmissionInformation.new(ori: find_ori)
+  end
+
+  def create
+    @info = AdditionalSubmissionInformation.new(info_params)
+    @info.ori = find_ori
+
+    if @info.save
+      redirect_user
+    else
+      render :new
+    end
+  end
+
+  private
+
+    def info_params
+      params.require(:additional_submission_information)
+            .permit(:submission_year,
+                    :k_9_officer_severirly_injured_count,
+                    :non_sworn_uniformed_injured_or_killed_count,
+                    :civilians_injured_or_killed_by_non_sworn_uniformed_count)
+    end
+
+    def find_ori
+      @ori = if @current_user.allowed_oris.include?(params[:ori])
+               params[:ori]
+             else
+               @current_user.ori
+             end
+    end
+
+    def redirect_user
```

Comment:
see screenshot. redirect is to handle between users with single ori vs multiple oris.
If all the oris are completed they are redirected to dashboard. Else, when there are multiple oris and not all are completed, it redirects back to the voluntary submission form.

<img width="1552" alt="screenshot 2016-12-01 10 12 12" src="https://cloud.githubusercontent.com/assets/1409179/20806178/b3d5cb52-b7ae-11e6-950c-85898063aeb1.png">